delta/sourcing-psql/src/test/scala/ch/epfl/bluebrain/nexus/delta/sourcing/EntityDependencyStoreSuite.scala

```diff
@@ -109,7 +113,7 @@ class EntityDependencyStoreSuite extends BioSuite with Doobie.Fixture {
       )
   }
 
-  test("Fetch values for direct dependencies of id1") {
+  test("Fetch latest state values for direct dependencies of id1") {
```

Comment:
This doesn't check that the latest state is returned, as both the states appear the same in the result (so you could be returning the tagged version rather than latest)

To communicate the intent of the test better, it would be nice to express "I want the latest version of state 2, and I don't want the other version of state 2"